rsts/deployment/sandbox.rst

```diff
@@ -186,32 +102,39 @@ We'll proceed like with :ref:`locally hosted flyte <deploy-sandbox-local>` with
 Flyte configuration on your remote cluster.
 
 
-#. The Flyte sandbox can be deployed via a helm chart. From the root dir of the flyte repo run ::
+#. Add Helm repo for flyte ::
 
-    helm repo add flyte https://flyteorg.github.io/flyte
-    helm install -n flyte -f values.yaml --create-namespace flyte flyte/flyte
+    helm repo add flyteorg https://helm.flyte.org
 
-#. For customizations instructions, see ``/charts/flyte/README.md`` in the flyte repo.
+#. Install Flyte dependency helm chart (this will install the minio, Postgres, Kubernetes-dashboard, and contour) ::
 
-#. You can now port-forward (or if you have load-balancer enabled then get an LB) to connect to remote FlyteConsole, as follows ::
+    helm install -n flyte flyte-deps flyteorg/flyte-deps --create-namespace --set webhook.enabled=false,minio.service.type=LoadBalancer,contour.enabled=true,contour.envoy.service.type=LoadBalancer,kubernetes-dashboard.service.type=LoadBalancer
```

Comment:
Why do we need to say "webhook.enabled=false" for the flyte-deps helm chart?
Should the others be moved to a values-sandbox.yaml inside flyte-deps helm chart?

Reply:
webhook exist in both chart flyte-dep and flyte-core. We need to disable one

Reply:
I don't get it. I thought the deps helm chart doesn't install any flyte components.. what webhook is this?

Reply:
In sandbox lite we use flyte-dep and that needed webhook deployment

Reply:
aha, it's just the external service file: https://github.com/flyteorg/flyte/blob/master/charts/flyte-deps/templates/webhook/service.yaml... alright, I get it...

? Should the others be moved to a values-sandbox.yaml inside flyte-deps helm chart?

Reply:
meaning?

Reply:
> Should the others be moved to a values-sandbox.yaml inside flyte-deps helm chart?

meaning ?